### ISSUE TITLE
check for honeypot/transfer lock

### ISSUE THREAD

**Issue by playtribes1** [Blockys69/ARBITRUM-SNIPING-BOT-ASTRA-GUI#1]
also would be nice to have a few additional dexes on arbitrum... any way to contact you about this?

**Comment by Blockys69** [comment]
Yo will let you know

**Comment by miketradesdefi** [comment]
in the install directions, what does it mean to " Open terminal in the bot folder"? What terminal? Im on MacOs is that the application or something located in the bot folder?

**Comment by miketradesdefi** [comment]
> Yo will let you know

in the install directions, what does it mean to " Open terminal in the bot folder"? What terminal? Im on MacOs is that the application or something located in the bot folder?

**Comment by Blockys69** [comment]
Yeah man, you open the folder in Visual Studio Code and then you click on "Terminal" and "New Terminal in the Folder"

**Comment by miketradesdefi** [comment]
> Yeah man, you open the folder in Visual Studio Code and then you click on "Terminal" and "New Terminal in the Folder"

I dragged the folder into VisualCode but I don't see any terminal options. Also, i downloaded python 3.11 does that have anything to do with it? I see in another video you must use 3.7.

**Comment by Blockys69** [comment]
Python should not be an issue as long as it is 3+ but you can definitely update to 3.7.

You drag the folder into VS and at the top menu you can see "Terminal" or you right click on the folder in VS and do "Open Terminal here"

**Comment by miketradesdefi** [comment]
ok lol got that part, thanks. Now when I type "pip install -r requirements.txt" it says (bash: pip: command not found)

**Comment by miketradesdefi** [comment]
how does the python download play a part with the VS application?

**Comment by Blockys69** [comment]
Python 3.7 should have the pip included so go with pip3 install -r requirements.txt

**Comment by miketradesdefi** [comment]
> Python 3.7 should have the pip included so go with pip3 install -r requirements.txt
 
Ok that worked, but the "python3 astra-arbitrum.py" line did not work. It says -

"Traceback (most recent call last):
  File "astra-arbitrum.py", line 84, in <module>
    from web3 import Web3 as R
ModuleNotFoundError: No module named 'web3'

**Comment by Blockys69** [comment]
Try doing pip3 install web3

**Comment by Blockys69** [comment]
> > Python 3.7 should have the pip included so go with pip3 install -r requirements.txt
> 
> Ok that worked, but the "python3 astra-arbitrum.py" line did not work. It says -
> 
> "Traceback (most recent call last): File "astra-arbitrum.py", line 84, in from web3 import Web3 as R ModuleNotFoundError: No module named 'web3'

This shows me you are not installing the requirements in the folder. You need to have the document location written in the terminal, otherwise your terminal is not open in the folder and you are not installing the modules at all

**Comment by miketradesdefi** [comment]
I downloaded the arbitrum zip file from github onto my desktop then unzipped the file. Also unzipped the ABI and Theme zip into the same folder. then dragged the folder into VS. Was there anything else specifically I should do? Thanks

**Comment by Blockys69** [comment]
The terminal is the problem, it always opens in the home folder, but you need to open the terminal in the bot folder directly otherwise you are installing web3 to your home folder :D

You right click on the left sidebar where your folder is open and click "Open in Integrated Terminal" and then install requirements.txt

**Comment by miketradesdefi** [comment]
<img width="446" alt="image" src="https://user-images.githubusercontent.com/126417316/221435307-3c9f76c8-31bf-4ede-bb1f-5ecdddb2102a.png">

I have the folder open, I know I'm looking for an open terminal option, but I right-clicked on the folder and it only gives these options

**Comment by miketradesdefi** [comment]
Initially, I did use the open in integrated terminal" option I clicked in the blank white space on the left-hand section, and ran the first requirements line, which worked, but the python3 didn't. Thanks for being patient with me btw

**Comment by miketradesdefi** [comment]
Btw it also says this when i first open terminal: 
The default interactive shell is now zsh.
To update your account to use zsh, please run `chsh -s /bin/zsh`.
For more details, please visit https://support.apple.com/kb/HT208050.

Does this effect anything?

**Comment by miketradesdefi** [comment]
> The terminal is the problem, it always opens in the home folder, but you need to open the terminal in the bot folder directly otherwise you are installing web3 to your home folder :D
> 
> You right click on the left sidebar where your folder is open and click "Open in Integrated Terminal" and then install requirements.txt

So click on any random file in the folder then "click open in integrated terminal"?

**Comment by miketradesdefi** [comment]
Ok I understand what you are saying now, the terminal keeps opening in the home folder in the bottom section, is it supposed to open up in a separate window instead?

**Comment by Blockys69** [comment]
Hmm, this is getting kinda complicated :D

So can you send me a screenshot of the entire VS please

**Comment by miketradesdefi** [comment]
Finally got it, i had to download python 3.7.2 and also update the pip in the terminal

**Comment by miketradesdefi** [comment]
Thanks for your help, now I will play around with it. Any tips for Arbitrum? Does it have support for all Dexes or just Sushiswap?

**Comment by Blockys69** [comment]
Sushiswap for now as all tokens launch on it, but will add zyber :)

snipe with little and then when you understand the stop loss snipe big haha

**Comment by miketradesdefi** [comment]
> Sushiswap for now as all tokens launch on it, but will add zyber :)
> 
> snipe with little and then when you understand the stop loss snipe big haha

If it's not a burden, consider Camelot, SharkySwap, and OreoSwap! Of those three Camelot is the biggest Arbitrum native dex with an integrated launchpad. So I think it would be great for this tool. Again thanks for your help

**Comment by miketradesdefi** [comment]
Thanks, send me a message on Telegram - miketradesdefi! I research new projects full time I'll let you know if I find anything worth sniping!
